documentation/style-guide.rst

```diff
@@ -88,6 +88,15 @@ Unix
    1970s.
 
 
+Use simple language
+===================
+
+Avoid esoteric phrasing where possible.  Our audience is world-wide and may not
+be native English speakers.
+
+Don't use Latin abbreviations like "e.g." or "i.e." where English words will do.
```

Comment:
... such as "for example" and "that is" ?

Reply:
done